
# folderPath: /home/jl/pra_data/202057_12_35/auto
# picName: building.jpg

autoSpeedMax: 2.0
manualSpeedMax: 1.0
omegaMax: 1.5


